_episodes/08-adding-ci-to-your-code.md

```diff
@@ -71,7 +71,7 @@ The compilation will result in an output binary called `skim`.
 <!--![CI/CD Two Parallel Jobs]({{site.baseurl}}/fig/ci-cd-two-parallel-jobs.png)-->
 
 ## Check jobs
-
+>## Remember `act` is optional.
```

Comment:
I think this needs the `{: . callout}` and a line below for the content of the box , right?

Reply:
Also, probably add a `,` ("Remember, ")